SEED-platform/seed
# Refactor/analysis property display field

#### Any background context you want to provide?
See ticket

#### What's this PR do?
- use the organization settings property display field when identifying properties in analysis lists

#### How should this be manually tested?
- run an analysis; play around with the org settings for display field; you should see it update

#### What are the relevant tickets?

#2969 

#### Screenshots (if appropriate)
E.g. I changed the display field to be the Created Date
<img width="763" alt="Screen Shot 2021-11-02 at 3 57 28 PM" src="https://user-images.githubusercontent.com/18518728/139957682-0bfb5f67-77f9-474b-84ac-53f0098c9ecf.png">

<img width="763" alt="Screen Shot 2021-11-02 at 3 58 03 PM" src="https://user-images.githubusercontent.com/18518728/139957690-9befb745-d58c-465f-b60f-c4715a21915a.png">
